WORKSPACE

```diff
@@ -1,35 +1,19 @@
 workspace(name = "jsonnet")
 
-# This local_repository looks silly but it makes io_bazel_rules_jsonnet use
-# _this_ jsonnet, not another downloaded copy.
-local_repository(
-    name = "jsonnet",
-    path = ".",
-)
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
 
 git_repository(
     name = "io_bazel_rules_jsonnet",
     commit = "09ec18db5b9ad3129810f5f0ccc86363a8bfb6be",
```

Comment:
This commit does not work with bazel 0.24 anymore so could you please update to the latest commit `fd484046f9c4a32f4d696f05578907162d5a631f` and also add `shallow_since = "1551169313 +0100",` to obtain the canonical reproducible form.